src/formatDate.js

```diff
@@ -50,7 +50,42 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separator = fromFormat[3];
+  const newJointer = toFormat[3];
+  const result = [];
+
+  const splitDate = date.split(separator);
+
+  const obj = {};
+
+  for (let i = 0; i < splitDate.length; i++) {
+    obj[fromFormat[i]] = splitDate[i];
+  }
+
+  if (fromFormat.includes('YY') && toFormat.includes('YYYY')) {
+    const tmp = obj.YY;
```

Comment:
what is 'tmp'? make more understandable variable.
```suggestion
    const temporary = obj.YY;
```